app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ReservationModel.java

```diff
@@ -0,0 +1,70 @@
+package com.mobile.tandil.javabaseproject.mvp.model;
```

Comment:
please review the logic and the flow(the responsabilty of this logic)

Reply:
Done